specification/cosmos-db/resource-manager/readme.md

```diff
@@ -27,9 +27,27 @@ These are the global settings for the Cosmos-DB API.
 ``` yaml
 title: CosmosDBManagementClient
 openapi-type: arm
-tag: package-2023-04
+tag: package-preview-2023-03-15
```

Comment:
why this is allowed ? we should never change a default tag to use an older preview version replace the newer version stable.

Reply:
@JeffreyRichter @mikekistler I think for this case, we have not reviewed the cross version breaking change yet,  because they are using a older preview to replace a newer version stable, which is the bind zone of the breaking change detection tool.

Reply:
oops, it seems we have released this version for all languages.

Reply:
Yeah, we shouldn't have.

Reply:
I'm not sure if you're asking Mike or I to do something here or not?
If you are, then I think we need more of an explanation as to what happened and what you want us to do.